.jshintignore

```diff
@@ -1 +1,2 @@
 node_modules
+lib/parser.js
```

Comment:
I added a newline so that the "no newline" mark is gone, but it seems GitHub still displays it as if there is no newline at the end. Should I add another?